contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -499,13 +455,40 @@ class IndexedDBResource {
     if (filterFn) {
       collection = collection.filter(filterFn);
     }
+    if (paginationActive) {
+      let results;
+      if (sortBy) {
+        // If we still have a sortBy value here, then we have not sorted using orderBy
+        // so we need to sort here.
+        if (reverse) {
+          collection = collection.reverse();
+        }
+        results = (await collection.sortBy(sortBy)).slice(0, maxResults + 1);
+      } else {
+        results = await collection.limit(maxResults + 1).toArray();
+      }
+      const hasMore = results.length > maxResults;
+      return {
+        results: results.slice(0, maxResults),
+        more: hasMore
+          ? {
+              ...params,
+              lft__gt: results[maxResults - 1].lft,
+            }
+          : null,
+      };
+    }
     if (sortBy) {
       if (reverse) {
         collection = collection.reverse();
       }
       collection = collection.sortBy(sortBy);
     }
-    return paginator.paginate(collection);
+    return collection.toArray();
+  }
+
+  whereLiveQuery(params = {}) {
```

Comment:
why is liveQuery needed here?

Reply:
It's not - I started trying to implement the pagination stuff using liveQuery to make sure it would be compatible, then realized I was doing more than was necessary for the immediate fix - then left these helper functions in here as generally useful, when I had originally intended to separate them out into a separate PR.

Reply:
I left this in, because it does no harm, and it's useful tiny flyby refactor.